tests/test-core/utils/js/ConditionalTestUtils.js

```diff
@@ -83,43 +83,30 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         });
     };
 
-    // Adds context awarenes of the browser's reported platform (operating
-    // system) for aiding in executing platform-specific tests, or skipping
-    // those known to fail on a particular platform due to lack of browser
-    // support for features
-
     // Checks string-based contextAwareness check values to see if they
     // contain the searchValue string
     fluid.test.conditionalTestUtils.contextValueContains = function (searchValue, checkValue) {
         var value = fluid.contextAware.getCheckValue(fluid.rootComponent, checkValue);
         return value.indexOf(searchValue) >= 0;
     };
 
-    fluid.test.conditionalTestUtils.getPlatformName = function () {
-        return navigator.platform ? navigator.platform : undefined;
-    };
-
-    // Functions for platform reporting for makeChecks
-    fluid.test.conditionalTestUtils.isLinux = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Linux", "{fluid.platform}");
+    // Functions for browser platform reporting for makeChecks
+    fluid.test.conditionalTestUtils.isBrowserLinux = function () {
+        return fluid.test.conditionalTestUtils.contextValueContains("Linux", "{fluid.browser.platformName}");
     };
 
-    fluid.test.conditionalTestUtils.isMac = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Mac", "{fluid.platform}");
+    fluid.test.conditionalTestUtils.isBrowserMac = function () {
+        return fluid.test.conditionalTestUtils.contextValueContains("Mac", "{fluid.browser.platformName}");
     };
 
-    fluid.test.conditionalTestUtils.isWindows = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Windows", "{fluid.platform}");
+    fluid.test.conditionalTestUtils.isBrowserWindows = function () {
```

Comment:
@waharnum i'm struggling a bit with this in my head, but I think that it might be better to call these something like "isBrowserOnWindows". I think that's more specific, and in this particular case less ambiguous as it might be confused with the browsers window or window object. What do you think?